To fix Issue 223, the '<module>' in the code fragment below 
should be in a monospaced font.

.. code-block:: pycon

    >>> raise Exception("Texto")
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    Exception: Texto

